src/MicrosoftAuthentication.js

```diff
@@ -0,0 +1,21 @@
+const MicrosoftAuthFlow = require('./MicrosoftAuthFlow')
+
+/**
+* Authenticates to Minecraft with Microsoft.
+*
+* @function
+* @param {object} options - Client Options
```

Comment:
this isn't how you jsdoc an object, take a look at https://jsdoc.app/tags-typedef.html